File: Beecrowd/1 - Beginner/2003 - 2176/2029 - Honey Reservoir/C++/2029_HoneyReservoir.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double volume, diameter;

    while (cin >> volume >> diameter) {
        double radius = diameter / 2;
        double area = 3.14 * radius * radius;
        double height = volume / area;

        cout << "ALTURA = " << fixed << setprecision(2) << height << endl;
        cout << "AREA = " << fixed << setprecision(2) << area << endl;
    }

    return 0;
}

File: Beecrowd/1 - Beginner/2003 - 2176/2029 - Honey Reservoir/2029_HoneyReservoir.py
import sys

for line in sys.stdin:
    volume = float(line)
    diameter = float(input())
    
    radius = diameter / 2
    area = 3.14 * radius * radius
    height = volume / area

    print(f"ALTURA = {height:.2f}")
    print(f"AREA = {area:.2f}")
